db/models/ally.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Ally = db.define('allies', {
+  username: Sequelize.STRING,
+  phone_number: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  email: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      isEmail: true
+    }
+  },
+  job_title: Sequelize.STRING,
+  company: Sequelize.STRING,
+  industry: Sequelize.ARRAY(Sequelize.STRING),
+  max_clients: Sequelize.INTEGER
+}, {
+  indexes: [
+    { unique: false, fields: ['industry'] }
```

Comment:
I think we were planning on trying to match women with advocates from a similar industry so this presents an easy way to do that. It may be unnecessary, or maybe there is a better way?